twisted/conch/client/default.py

```diff
@@ -156,21 +161,25 @@ def _ebSetAgent(self, f):
         userauth.SSHUserAuthClient.serviceStarted(self)
 
     def _getPassword(self, prompt):
-        try:
-            oldout, oldin = sys.stdout, sys.stdin
-            sys.stdin = sys.stdout = open('/dev/tty','r+')
-            p=getpass.getpass(prompt)
-            sys.stdout,sys.stdin=oldout,oldin
-            return p
-        except (KeyboardInterrupt, IOError):
-            print()
-            raise ConchError('PEBKAC')
+        with self._replaceStdoutStdin():
+            try:
+                p=getpass.getpass(prompt)
```

Comment:
I wish twistedchecker were working; sorry for the nitpick.  There should be spaces around the equal sign.